scheduler/scheduler-smartproxy/src/main/java/org/ow2/proactive/scheduler/smartproxy/SmartProxyImpl.java

```diff
@@ -205,9 +192,18 @@ public static synchronized SmartProxyImpl getInstance() {
     }
 
     @Override
-    public void init(String url, String user, String pwd) throws SchedulerException, LoginException {
-        CredData cred = new CredData(CredData.parseLogin(user), CredData.parseDomain(user), pwd);
-        init(url, cred);
+    public void init(String url, String user, String pwd, File credentialFile, boolean insecure) throws SchedulerException, LoginException {
+        if (credentialFile != null) {
+            try {
+                Credentials credentials = Credentials.getCredentials(credentialFile.getAbsolutePath());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "credentials" which hides the field declared at line 138. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AHiddenFieldCheck)